src/cordova/app.cordova.ts

```diff
@@ -10,7 +10,9 @@ import { getCurrentSettings, initSecureStorage, storeKeys } from "./storage"
 import { bioAuthenticate, isBiometricAuthAvailable } from "./bio-auth"
 
 const iframe = document.getElementById("walletframe") as HTMLIFrameElement
-const showSplashScreenOnIOS = () => (process.env.PLATFORM === "ios" ? navigator.splashscreen.show() : undefined)
+const showSplashScreenOnIOS = () => (cordova.platformId === "ios" ? navigator.splashscreen.show() : undefined)
```

Comment:
Let's better add `PLATFORM=<platform>` to the `dev` npm scripts in `cordova/package.json`. Just for consistency's sake, so we don't need to think about when to use which one ;)

Reply:
It actually is defined in the scripts, I looked it up before changing it this way but it does not work anyway.

I checked: If you run `dev:android` or `dev:ios` then `process.env.PLATFORM` will be defined within the application code but not within our cordova code. 
I could not come up with a solution for this maybe you have an idea what could be the reason behind this?